DDL.sql

```diff
@@ -0,0 +1,263 @@
+CREATE TABLE bikeConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    distanceCovered INTEGER NOT NULL
+);
+
+CREATE TABLE parts(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    weight FLOAT NOT NULL
+);
+
+CREATE TABLE BMSConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    totalVoltageUnits TEXT DEFAULT NULL,
+    batteryVoltageUnits TEXT DEFAULT NULL,
+    currentUnits TEXT DEFAULT NULL,
+    thermistorTempUnits TEXT DEFAULT NULL,
+    packTempUnits TEXT DEFAULT NULL,
+    bqTempUnits TEXT DEFAULT NULL,
+    cellVoltageUnits TEXT DEFAULT NULL
+);
+
+CREATE TABLE IMUConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    vectorUnits TEXT DEFAULT NULL
+);
+
+CREATE TABLE TMUConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    thermUnits TEXT NOT NULL
+);
+
+CREATE TABLE TMSConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    sensorTempUnits TEXT NOT NULL,
+    pumpSpeedUnits TEXT NOT NULL,
+    fanSpeedUnits TEXT NOT NULL
+);
+
+CREATE TABLE PVCConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE BikeBoardConfig(
+    bikeConfigID INTEGER,
+    boardConfig INTEGER,
+    PRIMARY KEY ( bikeConfigID, boardConfig)
+);
+
+CREATE TABLE BikePartConfig(
+    ID SERIAL PRIMARY KEY,
+    bikeConfigId INTEGER NOT NULL,
+    partNumber TEXT NOT NULL
+);
+
+CREATE TABLE Context(
+    ID SERIAL PRIMARY KEY,
+    date DATE NOT NULL,
```

Comment:
`date` is a reserved keyword in SQL
```suggestion
    contextDate DATE NOT NULL,
```

Reply:
Fixed.